pandas/tests/indexes/multi/test_get_set.py

```diff
@@ -329,3 +329,21 @@ def test_set_levels_with_iterable():
         [expected_sizes, colors], names=["size", "color"]
     )
     tm.assert_index_equal(result, expected)
+
+
+def test_at_indexing_fails_multiindex():
+    # GH9259
+    cols = [("a_col", chr(i + 65)) for i in range(2)]
+    idx = [("a_row", chr(i + 65)) for i in range(2)]
+    df = pd.DataFrame(
+        np.linspace(1, 4, 4).reshape(2, 2),
+        index=pd.MultiIndex.from_tuples(idx),
+        columns=pd.MultiIndex.from_tuples(cols),
+    )
+    s = pd.Series(np.linspace(1, 2, 2), index=pd.MultiIndex.from_tuples(idx))
```

Comment:
nitpick: can you call this `ser`?  1-letter variable names are a hassle to grep for

Reply:
could also just define it `ser = df[("a_col", "A")]`

Reply:
Done!